### DISCUSSION TITLE
23차 모임 (2024-09-10) 발표 주제 제안

### DISCUSSION THREAD

**Discussion by haksungjang** [OpenChain-KWG/meeting#7] [General]
안녕하세요. 

OpenChain Korea Work Group 23차 모임이 2024년 9월10일 오후1시부터 ETRI 서울사무소에서 예정되어 있습니다.

이 때 주제 발표를 원하시는 분은 발표 제목과 간단한 내용을 comment로 남겨주시기 바랍니다.
혹시, (내가 발표하지는 못하지만) 누군가가 공유해주기를 원하시는 주제가 있어도 comment 남겨주세요.

이번 모임에서는 주제 발표 세션과 더불어 Lightening Talk를 시도해보려고 합니다.
Lightening Talk는 특정 주제에 대해 3~5분 가량 가볍게 소개하는 시간입니다. Lightening Talk로 다루고자 하는 주제가 있다면 comment 남겨주시기 바랍니다.

감사합니다.

**Comment by haksungjang** [comment]
주제발표

가제: 글로벌 자동차 제조사가 요구하는 오픈소스 라이선스 준수 활동
 - 발표: 티맵모빌리티

**Reply by minjij0** [reply]
티맵모빌리티 발표 주제 추가합니다.

주제 : 
- 글로벌 자동차 제조사가 요구하는 오픈소스 관련 요건
- 소송 사례 공유 : Github Copilot / Google v. Oracle / Getty Images v Stability AI

발표자 : 티맵모빌리티 (김문엽 변호사)

발표시간 : 20분

**Comment by haksungjang** [comment]
Lightening Talk

'XZ Open Source Attack : 오픈소스 커뮤니티의 신뢰를 이용한 최초의 공급망 공격 사례'에 대한 소개
https://docs.google.com/document/d/1TGds3d9TErWdNcgC0Jw_CYrxpixXRhHl2KMiiVtWzQI/edit#heading=h.5fzr1w9pp1pq

카카오 황은경 파트장님께서 제안해 주셨습니다.

**Comment by haksungjang** [comment]
Lightening Talk

'Copilot 등 생성형 AI가 만들어내는 code를 저작권 이슈를 최소화하여 사용하는 방법'에 대해 가볍게 다뤄주실 분 계신가요?
또한, 최근 Copilot 소송 판결이 나왔는데 이에 대해서도 소개해 주실 분 계신가요? 

감사합니다.

**Comment by jiyoungyeon** [comment]
안녕하세요. 텔레칩스 연지영입니다. 
발표 신청합니다!  
주제: 오픈소스 라이선스 간 양립성 충돌 문제 조항과 판단 기준에 관한 연구(GPL 라이선스를 중심으로)
시간은 30분 정도 소요될 것 같습니다. 
감사합니다.

**Reply by haksungjang** [reply]
연지영 님! 발표 자원 감사합니다~ ^^

**Comment by haksungjang** [comment]
'온보딩 TF’ 운영 소개 - 이서연님(@syleeeee )

- 처음 접한 분들이 질문하는 채널을 미리 만들어서 공유하고, 히스토리 FAQ 형태로 제작
- FAQ 제작을 위한 자원(질문자/답변자/정리자)을 받아서 TF 운영

**Comment by haksungjang** [comment]
Education Work Group 신설 논의 - 박정숙 님 (@jungsp)

- 온보딩 TF를 정기적인 모임으로 확대하여 여러 교육 자료를 만드는 subgroup 결성
- 유용한 글로벌 자료 번역

**Reply by jungsp** [reply]
대장님, 이거 제가 준비하는 건가요?^^ 뭐 준비하라고 하셔도 충성!^^

**Comment by haksungjang** [comment]
'ETRI 오픈소스 거버넌스 소개'(가제) - 박정숙 님

박정숙 님, (@jungsp)
발표 주제와 시간을 댓글로 남겨주시면 고맙겠습니다. 

감사합니다.

**Reply by jungsp** [reply]
안녕하세요? ETRI 발표는 "ETRI 오픈소스 거버넌스 소개"가 고민해 봤을 때 가장 좋은 것 같습니다. 시간은 20분으로 잡아주시면 감사하겠습니다. 답이 늦어서 죄송합니다.^^
